tools/server/tests/unit/test_chat_completion.py

```diff
@@ -171,18 +171,18 @@ def test_apply_chat_template():
     assert res.body["prompt"] == "<|START_OF_TURN_TOKEN|><|SYSTEM_TOKEN|>You are a test.<|END_OF_TURN_TOKEN|><|START_OF_TURN_TOKEN|><|USER_TOKEN|>Hi there<|END_OF_TURN_TOKEN|><|START_OF_TURN_TOKEN|><|CHATBOT_TOKEN|>"
 
 
-@pytest.mark.parametrize("response_format,n_predicted,re_content", [
-    ({"type": "json_object", "schema": {"const": "42"}}, 6, "\"42\""),
-    ({"type": "json_object", "schema": {"items": [{"type": "integer"}]}}, 10, "[ -3000 ]"),
-    ({"type": "json_schema", "json_schema": {"schema": {"const": "foooooo"}}}, 10, "\"foooooo\""),
-    ({"type": "json_object"}, 10, "(\\{|John)+"),
-    ({"type": "sound"}, 0, None),
+@pytest.mark.parametrize("response_format,n_predicted,re_content,expected_status_code", [
+    ({"type": "json_object", "schema": {"const": "42"}}, 6, "\"42\"", 200),
+    ({"type": "json_object", "schema": {"items": [{"type": "integer"}]}}, 10, "[ -3000 ]", 200),
+    ({"type": "json_schema", "json_schema": {"schema": {"const": "foooooo"}}}, 10, "\"foooooo\"", 200),
+    ({"type": "json_object"}, 10, "(\\{|John)+", 200),
+    ({"type": "sound"}, 0, None, 400),
     # invalid response format (expected to fail)
-    ({"type": "json_object", "schema": 123}, 0, None),
-    ({"type": "json_object", "schema": {"type": 123}}, 0, None),
-    ({"type": "json_object", "schema": {"type": "hiccup"}}, 0, None),
+    ({"type": "json_object", "schema": 123}, 0, None, 400),
+    ({"type": "json_object", "schema": {"type": 123}}, 0, None, 400),
+    ({"type": "json_object", "schema": {"type": "hiccup"}}, 0, None, 400),
```

Comment:
I think these changes are quite redundant, just need to change `assert res.status_code != 200` to `assert res.status_code == 400`